core/src/cf/mutations.rs

```diff
@@ -61,29 +62,52 @@ impl Default for DatabaseMutation {
 		Self::new()
 	}
 }
+
 // Change is a set of mutations made to a table at the specific timestamp.
-#[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Serialize, Deserialize, Store, Hash)]
 #[revisioned(revision = 1)]
+#[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Serialize, Deserialize, Store, Hash)]
 pub struct ChangeSet(pub [u8; 10], pub DatabaseMutation);
 
 impl TableMutation {
 	pub fn into_value(self) -> Value {
-		let (k, v) = match self {
-			TableMutation::Set(_t, v) => ("update".to_string(), v),
-			TableMutation::SetPrevious(_t, Value::None, v) => ("create".to_string(), v),
-			TableMutation::SetPrevious(_t, _previous, v) => ("update".to_string(), v),
+		let mut h = BTreeMap::<String, Value>::new();
+		let h = match self {
+			TableMutation::Set(_thing, v) => {
+				// if true { panic!("The previous value is {:?},     {:?}", _thing, v); }
+				h.insert("update".to_string(), v);
+				h
+			}
+			TableMutation::SetWithDiff(_thing, Value::None, _operations) => {
+				h.insert("original".to_string(), Value::None);
+				h.insert("create".to_string(), Value::Array(Array(Vec::new())));
+				h
+			}
+			TableMutation::SetWithDiff(_thing, current, operations) => {
+				// TODO
+				// TODO -- Figure this out
```

Comment:
It exists as Value::patch

Reply:
This is for storing, so since we are storing the diffs we should be good...? EIther way, it does not get applied here. This is end stage of translation in:
tx_change -> TableMutation -> Value -> KV.store(value)